README.md

```diff
@@ -24,15 +24,16 @@
 
    ```xml
    <?xml version="1.0"?>
-   <ruleset name="Webimpress Coding Standard"
+   <ruleset name="MyCodingStandard"
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
             xsi:noNamespaceSchemaLocation="./vendor/squizlabs/php_codesniffer/phpcs.xsd">
-       <rule ref="./vendor/webimpress/coding-standard/ruleset.xml"/>
+
+       <rule ref="WebimpressCodingStandard"/>
 
        <!-- Paths to check -->
-       <file>config</file>
-       <file>src</file>
-       <file>test</file>
+       <file>config/</file>
+       <file>src/</file>
+       <file>test/</file>
```

Comment:
I guess these are not really needed?

Reply:
Not really. They show that for example `src` is a directory, not a file.